### DISCUSSION TITLE
Align text in table header cells

### DISCUSSION THREAD

**Discussion by plakhin** [filamentphp/filament#4621] [Ideas]
It would be nice to be able to align text in table header cells. I'm using `->extraAttributes(['class' => 'text-right']),` for columns that has numbers and it would be nice to have header aligned the same. Also sorting button (arrow) should be displayed before the text in the header when header text aligned right.

**Comment by danharrin** [comment]
I think we have `extraHeaderAttributes()` ?

**Comment by plakhin** [comment]
Oh, there's `->alignRight()` method that aligns header text to the right. Missed it in the docs somehow.

**Reply by danharrin** [reply]
Nice

**Reply by cawecoy** [reply]
@plakhin  `->alignRight()` won't align the header, but column content. Has it changed?

**Reply by danharrin** [reply]
I think they should both be aligned at once IIRC
